File: BkG1UU/very_big_sum.py
def a_very_big_sum(ar):
    sum = 0
    
    for i in ar:
        sum+=i

    print(sum)

list = [1000000001, 1000000002, 1000000003, 1000000004, 1000000005]
a_very_big_sum(list)

File: BkG1UU/very_big_sum.cpp
#include <iostream>
#include <vector>

long aVeryBigSum(std::vector<long> ar)
{
    unsigned long sum{ 0 };
    for (auto i : ar) sum+=i;
    return sum;
}

int main(int argc, char const *argv[])
{
    int i,n, num;
    std::vector<long> ar;

    std::cin >> n;

    for (i = 0; i < n; i++)
    {
        std::cin >> num;
        ar.push_back(num);
    }

    aVeryBigSum(ar);
    return 0;
}
